runtime/gc_vlhgc/IncrementalGenerationalGC.cpp

```diff
@@ -209,7 +211,7 @@ MM_IncrementalGenerationalGC::initialize(MM_EnvironmentVLHGC *env)
 	 * except it has not been hard coded in command line
 	 */
 	if (0 == extensions->tarokAllocationAgeUnit) {
-		extensions->tarokAllocationAgeUnit = extensions->tarokIdealEdenMaximumBytes;
+		extensions->tarokAllocationAgeUnit = oldIdealEdenMaximumBytes;
```

Comment:
could we pull up these 2 age related inits before SchedulingDelegate interferes, to avoid this temp variable?

Reply:
i'm even ok with leaving it initialized with the new value, better than this

Reply:
pull up these 2 age related inits before SchedulingDelegate interferes